update_prevalence.py

```diff
@@ -357,14 +357,19 @@ def partial_vaccination_total(self):
             return reduce(lambda x, key: x + self.vaccines_by_type[key].partial_vaccinations, self.vaccines_by_type, 0)
         return self.vaccines_total.partial_vaccinations
 
-    # Compute the estimated risk ratio for an unvaccinated person vs an average person.
-    # Average prevalence = sum(prevalence in group * proportion of population in group)
-    #                    = sum(vaccine_mult * unvaccinated_prev * proportion of population)
-    # Unvaccinated risk / Average risk = sum(vaccine_mult * proportion of pop)
-    #                                  = population / sum(vaccine_mult * proportion of pop)
-    # Where the sum is taken over all vaccine types and status (incl no vaccine)
+    # Compute the estimated risk ratio for an unvaccinated person vs an average person,
+    # so that we can convert average risk (which we know from prevalence data) to
+    # unvaccinated risk if we have sufficiently detailed vaccination information.
+    #
+    # risk_sum / population will be the average vaccine multiplier across the
+    # entire population, including unvaccinated individuals.
+    # If nobody were vaccinated, the average vaccine multiplier would be 1.
+    # Therefore
+    # unvaccinated_vaccine_multiplier / avg_vaccine_multiplier
```

Comment:
I find using `unvaccinated_vaccine_multiplier` here confusing because the vaccine multiplier for "no vaccine" is 1.
I was using `unvaccinated_relative_risk`, but we could call it something like `unvaccinated_prevalence_vs_average`.